src/app/api/participant.py

```diff
@@ -0,0 +1,39 @@
+from app.RatingCalculator import RatingCalculator
+from app.api.base import BaseAuthResource
+from app.participant.models import ParticipantModel, create_participant, delete_participant, update_participant
+from flask.ext.restful import marshal, fields
+
+__author__ = 'Alex'
+
+participant_template = {
+    'league_id': fields.String,
+    'participant_id': fields.String,
+    'name': fields.String,
+    'rating': fields.String
+}
+
+
+class ParticipantListAPI(BaseAuthResource):
+    def get(self, league_id):
+        participants = ParticipantModel.query(league_id=league_id).fetch()
+        return {'data': [marshal(1, participant_template) for 1 in participants]}
+
+    def post(self):
+        new_participant = create_participant(self.user.user_id, self.args.get('league_id'),
+                                             self.args.get('name'), rating=self.args.get('rating'))
+        return {'data': marshal(new_participant, participant_template)}
+
+
+class ParticipantAPI(BaseAuthResource):
```

Comment:
your route for this class is ``` '/participant/<string:league_id>/<string:participant_id>' ``` which means all your methods below will get the args league_id and participant_id (they must be in the signature even if you don't use them).